app.js

```diff
@@ -1,8 +1,64 @@
 // import functions and grab DOM elements
+// Cups
+const cup1 = document.getElementById('cup1');
+const cup2 = document.getElementById('cup2');
+const cup3 = document.getElementById('cup3');
 
-// initialize global state
+// Buttons
+const btn1 = document.getElementById('btn1');
+const btn2 = document.getElementById('btn2');
+const btn3 = document.getElementById('btn3');
+
+// Outputs
+const wins = document.getElementById('correct-guess');
+const losses = document.getElementById('incorrect-guess');
+const totals = document.getElementById('total-guess');
 
+// initialize global state
+let win = 0;
+let total = 0;
+const cups = [cup1, cup2, cup3];
+const buttons = [btn1, btn2, btn3];
```

Comment:
However, it appears you are not utilizing this buttons array.  Consider getting rid of unused variables.

Reply:
I thought I pushed that. Must have forgot. Thanks for reminding me. I merged that change.